tests/main/mounts-persist-refresh-content-snap/task.yaml

```diff
@@ -55,9 +56,10 @@ execute: |
   "$TESTSTOOLS"/snaps-state install-local test-snapd-content-slot
 
   # Signal to kill the loop
-  rm ~test/snap/test-snapd-desktop-layout-with-content/common/keep-running
+  rm -f ~test/snap/test-snapd-desktop-layout-with-content/common/keep-running
   wait "$pid"
 
   # Ensure that /usr/share/fonts/foo-font was never missing.
-  MATCH 'exited' ~test/snap/test-snapd-desktop-layout-with-content/common/status
-  NOMATCH 'foo-font missing' ~test/snap/test-snapd-desktop-layout-with-content/common/status
+  MATCH 'exited' <~test/snap/test-snapd-desktop-layout-with-content/common/status
+  test-snapd-desktop-layout-with-content.sh -c 'stat /usr/share/fonts/foo-font' | tee after
+  NOMATCH 'foo-font missing' <~test/snap/test-snapd-desktop-layout-with-content/common/status
```

Comment:
I'm wondering if for diagnosing problems it could be helpful to define a debug section of this task that `cat`s out `before`, `after`, and `/home/test/snap/test-snapd-desktop-layout-with-content/common/status`?

Reply:
Yup, I'll add that on Monday